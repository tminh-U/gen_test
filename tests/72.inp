9
16 8 11 9 1 4 3 4 10 
11
2 9
5 5
5 8
6 7
7 9
4 4
6 7
1 1
3 8
1 6
8 9
