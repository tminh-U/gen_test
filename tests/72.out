32
2
12
3
8
2
3
2
14
10
4
